app/models/publisher.rb

```diff
@@ -103,6 +103,32 @@ class Publisher < ApplicationRecord
 
   store_accessor :feature_flags, VALID_FEATURE_FLAGS
 
+  def self.uphold_creators
+    joins(:uphold_connection).
+      where('uphold_connections.country IS NULL').
+      where(selected_wallet_provider_type: 'UpholdConnection').
+      or(
+        joins(:uphold_connection).
+        where.not("uphold_connections.country ILIKE '#{UpholdConnection::JAPAN}'").
+        where(selected_wallet_provider_type: 'UpholdConnection')
+      )
```

Comment:
```suggestion
    joins(:uphold_connection).
      where(selected_wallet_provider_type: 'UpholdConnection').
      where.not("uphold_connections.country ILIKE '#{UpholdConnection::JAPAN}'").
      where(uphold_connections: { is_member: true})
```

I think this simplifies things a bit. Also we should be paying `is_member` this whole time

Reply:
You have to include the `or` or you won't pick up items where the country `IS NULL`.

Reply:
Made a ticket to track this https://github.com/brave-intl/publishers/issues/3243